build.gradle

```diff
@@ -132,14 +133,14 @@ test {
 }
 
 dependencies {
-    testCompile "org.scala-lang:scala-library:2.10.5"
-    testCompile "com.datastax.spark:spark-cassandra-connector-embedded_2.10:1.2.1"
-    testCompile "org.scalatest:scalatest_2.10:2.2.4"
+    testCompile "org.scala-lang:scala-library:$scalaVersion"
+    testCompile "com.datastax.spark:spark-cassandra-connector-embedded_$ScalaVersionShort:1.2.1"
```

Comment:
Just curious, why do we need an "embedded" connector dependency here?

Reply:
yeah we no longer need this i'll remove it